specs/lib/cacheOptimizeIndicesSpec.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var cacheOptimizeIndices = require('../../lib/cacheOptimizeIndices');
+var addDefaults = require('../../lib/addDefaults');
+var readGltf = require('../../lib/readGltf');
+var readAccessor = require('../../lib/readAccessor');
+
+var gltfPath = './specs/data/cacheOptimizeIndices/chinesedragon.gltf';
+
+describe('cacheOptimizeIndices', function() {
+    it('reorders indices', function(done) {
+        readGltf(gltfPath, function(gltf) {
+            addDefaults(gltf);
+            var indexAccessorId = gltf.meshes[Object.keys(gltf.meshes)[0]].primitives[0].indices;
+            var indicesBefore = readAccessor(gltf, gltf.accessors[indexAccessorId]);
+            
+            cacheOptimizeIndices(gltf);
+            
+            var indicesAfter = readAccessor(gltf, gltf.accessors[indexAccessorId]);
+            expect(indicesBefore).not.toEqual(indicesAfter);
```

Comment:
> should also check vertices since they may change for the pre-vertex shader cache.

Right now I believe I am only doing the post vertex cache optimization with tipsify. Another pull request will implement the pre vertex cache optimization